openmetadata-spec/src/main/resources/json/schema/entity/services/connections/testConnectionDefinition.json

```diff
@@ -20,10 +20,19 @@
           "description": "What is the goal of the step",
           "type": "string"
         },
+        "errorMessage": {
+          "description": "In case of error this message should be displayed on UI",
```

Comment:
We can specify here that this is something we handle manually on the test connection definition